lib/15/LibClang.jl

```diff
@@ -6937,7 +7112,7 @@ end
 
 const CINDEX_VERSION_MAJOR = 0
 
-const CINDEX_VERSION_MINOR = 62
+const CINDEX_VERSION_MINOR = 63
```

Comment:
although these low-level APIs are not exported by Clang.jl, but I think we should bump Clang.jl's minor version as well.

Reply:
Sure, fixed in 423820b.